src/parser.cu

```diff
@@ -418,6 +429,10 @@ int checkParameterValidity(
     dout(V_ERROR) << "At least 1 repetition is necessary!" << std::endl;
   }
 
+  if(adaptiveSteps < 1){
```

Comment:
Shouldn't we set some default value for adaptive steps ? The simple c example in the [README](https://github.com/ComputationalRadiationPhysics/haseongpu/blob/master/README.md#quick-c-application-ase-flux-example) specifies only the minimal number of rays per sample, thus, the example would fail with the new option.

Maybe do it like this:
* Set default to 1, if only min. rays was set
* Set default to 2, if min < max